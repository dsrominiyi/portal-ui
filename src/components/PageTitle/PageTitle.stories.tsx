import React from 'react';
import { storiesOf } from '@storybook/react';

import PageTitle from '.';

const stories = storiesOf('PageTitle', module);

const render = (text: string) => () => <PageTitle text={text} />;

export const storyRenders = {
  default: render('Page Title')
};

stories.add('default', storyRenders.default);
