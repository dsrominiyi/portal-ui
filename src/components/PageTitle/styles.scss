@import '../../style/vars';

.page-title {
  display: flex;
  font-size: 1.75rem;
  font-weight: 900;
  color: $brand-black;
  padding: 1.5rem 0.5rem 2rem 0.5rem;
}
