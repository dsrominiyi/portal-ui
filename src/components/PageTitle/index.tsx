import React, { FunctionComponent } from 'react';
import styles from './styles.scss';

interface Props {
  text: string;
}

export const PageTitle: FunctionComponent<Props> = ({ text }: Props) => (
  <div className={styles.pageTitle} test-id="page-title">
    {text}
  </div>
);

PageTitle.displayName = 'PageTitle';
export default PageTitle;
