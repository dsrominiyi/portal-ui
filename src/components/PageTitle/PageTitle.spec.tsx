import { storyRenders } from './PageTitle.stories';
import testStorySnapshots from '../../helpers/test/testStorySnapshots';

describe('PageTitle', () => {
  test('component story renders match snapshots', () => {
    testStorySnapshots(storyRenders);
  });
});
