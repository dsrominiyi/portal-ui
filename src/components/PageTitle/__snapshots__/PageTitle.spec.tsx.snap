// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle component story renders match snapshots: default 1`] = `
<div
  className="pageTitle"
  test-id="page-title"
>
  Page Title
</div>
`;
